src/ontology/cl-edit.owl

```diff
@@ -27808,6 +27815,72 @@ AnnotationAssertion(oboInOwl:hasAlternativeId obo:CL_4023154 "CL:0000328")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "ILX:0102311"^^xsd:string) rdfs:comment obo:CL_4023160 "Cartwheel cells share many of the features, molecular and electrophysiological, of cerebellar Purkinje neurons.")
+AnnotationAssertion(rdfs:label obo:CL_4023160 "cartwheel cell")
+SubClassOf(obo:CL_4023160 obo:CL_0000540)
+SubClassOf(obo:CL_4023160 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0002829))
+SubClassOf(obo:CL_4023160 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534))
+SubClassOf(obo:CL_4023160 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061537))
+SubClassOf(obo:CL_4023160 ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/uberon/core#synapsed_by> obo:CL_0000120))
+
+# Class: obo:CL_4023161 (unipolar brush cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:9193142"^^xsd:string) Annotation(oboInOwl:hasDbXref "WikipediaVersioned:Unipolar_brush_cell&oldid=1045937026"^^xsd:string) obo:IAO_0000115 obo:CL_4023161 "An excitatory glutamatergic interneuron found in the granular layer of the cerebellar cortex and also in the granule cell domain of the cochlear nucleus. Unipolar brush cells have a round or oval cell body with usually a single short dendrite that ends in a brush-like tuft of short dendrites unique to them known as dendrioles.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4023161 <http://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(rdfs:label obo:CL_4023161 "unipolar brush cell")
+SubClassOf(obo:CL_4023161 obo:CL_0000099)
+SubClassOf(obo:CL_4023161 ObjectSomeValuesFrom(obo:RO_0000053 obo:PATO_0070025))
+SubClassOf(obo:CL_4023161 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0002028))
+SubClassOf(obo:CL_4023161 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061535))
+
+# Class: obo:CL_4023162 (bushy cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "WikipediaVersioned:Bushy_cell&oldid=879061753"^^xsd:string) obo:IAO_0000115 obo:CL_4023162 "A neuron found in the anterior part of the ventral cochlear nucleus that has the appearance of bushes, having short dendrites. Bushy cells give outputs to different parts of the superior olivary complex.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4023162 <http://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(rdfs:label obo:CL_4023162 "bushy cell")
+SubClassOf(obo:CL_4023162 obo:CL_0000540)
+SubClassOf(obo:CL_4023162 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0002830))
+SubClassOf(obo:CL_4023162 ObjectSomeValuesFrom(obo:RO_0013001 obo:UBERON_0002128))
+
+# Class: obo:CL_4023163 (spherical bushy cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:31881018"^^xsd:string) Annotation(oboInOwl:hasDbXref "WikipediaVersioned:Bushy_cell&oldid=879061753"^^xsd:string) obo:IAO_0000115 obo:CL_4023163 "A bushy cell that receives only few large excitatory endbulb synapses from auditory nerves. Sperhical bush cells give excitatory input to the lateral and medial parts of the superior olive.")
```

Comment:
In definition, "Sperhical" appears to be a typo and should "Spherical bush cells give" be changed to singular form?